cli/crates/cli/tests/graphql-directive/snapshots/graphql_directive__namespaced-issue.snap

```diff
@@ -1,14 +1,14 @@
 ---
-source: crates/cli/tests/graphql-directive/main.rs
+source: cli/crates/cli/tests/graphql-directive/main.rs
 expression: "client.gql::<Value>(NAMESPACED_QUERY).variables(json!({ \"id\" : \"3\" })).await"
 ---
 data:
   gothub:
-    serverVersion: "1"
     pullRequestOrIssue:
       __typename: GothubIssue
-      title: Everythings fucked
       author:
         __typename: GothubUser
         email: pessimist@example.com
+      title: Everythings fucked
```

Comment:
use nicer wording here:)

Reply:
![2023-11-23_07-11-06](https://github.com/grafbase/grafbase/assets/13155277/ae9226f1-0c69-497e-8495-ee90bc64fa26)

Doing a quick round of fixes through the repo :p